TINY-9843: Fix border styles being split into longhand form after table updates (#8813)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

### Fixed
- Returning an empty string in custom context menu update function would result in a small white line appearing on right click and the native browser context menu would not show up. #TINY-9842
- Tab navigation incorrectly stopped around `iframe` dialog component. #TINY-9815

> - The border style of table was splitted into a longhand form after table updates, resulting a difference when being emitted during `getContent()`. #TINY-9843

Reviewer: I think we can keep it simple
```suggestion
- The border styles of a table were incorrectly split into a longhand form after table dialog updates. #TINY-9843
```

- If the selection contains more than one table cell, Quickbar toolbars are now positioned in the middle of the selection horizontally. #TINY-8297
- Exposed `dataTransfer` property of drag and drop events for elements with a `contenteditable="false"` attribute. #TINY-9601
- Screen readers now announce instructions for resizing the editor using arrow keys, when the resize handle is focused. #TINY-9793
- Dialog `tabpanel` tab labels are now allowed to word wrap for better readability with long labels #TINY-9947
- Added newlines before and after `details` elements in the output HTML. #TINY-9959
- Added padding for empty `summary` elements so that they can be properly edited. #TINY-9959

> - The `parse()` of `tinymce.html.Styles` API now accepts an optional `compressed` parameter to disable converting styles into shorthand form. #TINY-9843

Reviewer 2: ```suggestion
- `tinymce.html.Styles.parse()` API now accepts an optional `shouldCompress` parameter to disable converting styles into shorthand form. #TINY-9843
```

- Pressing the Backspace or Delete key when the insertion point was within a `details` element resulted in broken markup. #TINY-9884
- Making the selection into a list did not work if the selection included a block element with a `contenteditable="false"` attribute. #TINY-9823
- Inserting elements in the middle of the summary caused two summaries to appear within `details` elements. #TINY-9885

> - The border style of table elements were optimized into a shorthand form after updated through the dialog. #TINY-9843

Reviewer 2: ```suggestion
- The `border` style of table elements were optimized into shorthand form after being updated through the Table Properties dialog. #TINY-9843
```
